packages/react-components/react-components/src/Concepts/Icons/Icon.stories.mdx

```diff
@@ -25,8 +25,61 @@ You can import the sized icons in a similar way:
 import { AccessTime24Filled } from '@fluentui/react-icons';
 ```
 
+### List of available props
+
 Each icon also accepts `className` and `primaryFill` props for styling.
 
+| - Name -    | Description                                                                                                            | Default        |
+| ----------- | ---------------------------------------------------------------------------------------------------------------------- | -------------- |
+| bundleIcon  | Pass in the filled and unfilled version of an icon and this will combine them into one icon                            | -              |
```

Comment:
Is it a component prop?

Reply:
`bundleIcon` is not a component prop, rather a helper method. I've moved it from the prop table